### PR TITLE
docker: fix alignment of stopped images

### PR DESCRIPTION
**Pull request by andreasn**
Fixes issue #3905

### CODE REVIEW

**Comment by garrett** [comment]
This would make everything left-aligned, right? That would be really odd for file sizes. If it makes it all right-aligned, then please ignore. (But I thought there would need to be a CSS rule for `text-align: right` in a CSS file?)

**Comment by andreasn** [comment]
In my testing, the file sizes were already left aligned, but the "Stopped" label was right aligned.
How do you want the alignment?

**Comment by dperpeet** [comment]
@garrett @andreasn any updates on this?
I think file sizes would be easier to read when right-aligned, but I'll defer to your opinion after trying it out.

**Comment by garrett** [comment]
What I was suggesting was to right-align the file sizes, but left align the units, so that you can scan the numbers.

In most every time this is used in Cockpit, however, I think it's probably acceptable to just right-align and hope the width of the units line up. In most cases, you're comparing GBs to GBs and TBs to TBs... and not comparing either to KBs or Bs. Even still, the width of everything but B or bytes is probably close enough anyway.

The proper fix for right-aligning would be in the CSS files, of course. :wink: 

**TL;DR**: `text-align: right;` in the CSS for the column(s) of sizes works well enough.

**Comment by andreasn** [comment]
> TL;DR: text-align: right; in the CSS for the column(s) of sizes works well enough.

Sounds good to me.

**Comment by andreasn** [comment]
How does this look, @garrett ?

screenshot from 2016-05-25 15-29-43

**Comment by garrett** [comment]
Sure, that looks a lot better! Thanks.
